src/Data/CAS/RocksDB.hs

```diff
@@ -685,8 +712,8 @@ newCas
     -> Codec v
     -> Codec k
     -> [B.ByteString]
-    -> RocksDbCas v
-newCas db vc kc n = RocksDbCas $ newTable db vc kc n
+    -> IO (RocksDbCas v)
```

Comment:
Making this operation an `IO` action may cause issues in chainweb.

Reply:
It seemed fine in chainweb-node, because most uses of `newCas` coincided with resources being allocated.